pulsar-io/jdbc/core/src/main/java/org/apache/pulsar/io/jdbc/JdbcSinkConfig.java

```diff
@@ -76,15 +76,24 @@ public class JdbcSinkConfig implements Serializable {
     @FieldDoc(
         required = false,
         defaultValue = "500",
-        help = "The jdbc operation timeout in milliseconds"
+        help = "Enable batch mode by time. After timeoutMs milliseconds the operations queue will be flushed."
```

Comment:
what about introducing a new "batchTimeoutMs" ?

the timeoutMs parameter already existed

Reply:
`timeoutMs` has the same meaning as before. It doesn't have a meaningful name and that's why I updated the help.